dali/pipeline/operators/crop/bbox_crop.cc

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) 2017-2018, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "dali/pipeline/operators/crop/bbox_crop.h"
+
+namespace dali {
+
+DALI_SCHEMA(RandomBBoxCrop)
+    .DocStr(
+        R"code(Perform a prospective crop to an image while keeping bounding boxes consistent. Inputs must be supplied as two Tensors:
+        `Images` containing image data in NHWC format, and `BBoxes` containing bounding boxes represented as `[l,t,r,b]`.
+        Resulting prospective crop is provided as two Tensors: `Begin` containing the starting coordinates for the `crop` in `(x,y)` format,
+        and 'Size' containing the dimensions of the `crop` in `(w,h)` format. Bounding boxes are provided as a `(m*4)` Tensor,
+        where each bounding box is represented as `[l,t,r,b]` or `[x,y,w,h]`.)code")
+    .NumInput(2)
+    .NumOutput(3)
+    .AllowMultipleInputSets()
```

Comment:
Operators with more than one input won't work with AllowMultipleInputSets().

Each of input is present in workspace as Input(i).
If you have only one input `NumInput(1)`, this will allow to accept list of inputs as that one input, and map elements of those list to consecutive `Input(i)`. 
That way you will have a clash between one of the regular inputs and those additional from AllowMultipleInputSets.

Reply:
accepted